internal/recipe/local_spark.go

```diff
@@ -0,0 +1,66 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package recipe
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"io"
+	"os"
+	"testing"
+
+	"golang.org/x/xerrors"
+
+	"github.com/testcontainers/testcontainers-go/modules/compose"
+)
+
+//go:embed docker-compose.yml
+var composeFile []byte
+
+func Start(t *testing.T) error {
+	if _, ok := os.LookupEnv("AWS_S3_ENDPOINT"); ok {
```

Comment:
here is a check that allows to run tests with container running in a separate process (as it done currently in ci). 
cc: @zeroshade

Reply:
ah, very cool